src/modules/toggle_completed.js

```diff
@@ -0,0 +1,17 @@
+const toggleCompleted = (toDoList, taskManager, index) => {
+  const toDoItem = toDoList.querySelector(`[data-index='${index}']`);
+  if (!toDoItem) {
+    return;
+  }
```

Comment:
Even this piece of your code works fine, when using arrow functions, we can return implicitly if the function has a single line/statement without using the return keyword. We could return more complex values implicitly.